content/configuring/anomaly_scoring.md

```diff
@@ -0,0 +1,151 @@
+
+### Anomaly Score Thresholds
+
+An anomaly score threshold is the cumulative anomaly score at which an inbound request or an outbound response will be blocked.
+
+Most detected inbound threats carry an anomaly score of 5 (by default), while smaller violations, e.g. protocol and standards violations, carry lower scores. An anomaly score threshold of 7, for example, would require multiple rule matches in order to trigger a block (e.g. one "critical" rule scoring 5 plus a lesser-scoring rule, in order to reach the threshold of 7). An anomaly score threshold of 10 would require at least two "critical" rules to match, or a combination of many lesser-scoring rules. **Increasing the anomaly score thresholds makes the CRS less sensitive** and hence less likely to block transactions.
+
+{{% notice warning %}}
+Increasing the anomaly score thresholds may allow some attacks to bypass the CRS rules.
+{{% /notice %}}
+
+{{% notice tip %}}
+A common practice when working with a **new** CRS deployment is to start with very high anomaly score thresholds (even as high as 10000) and gradually lower them over time, through multiple iterations of a tuning process.
+{{% /notice %}}
+
+CRS uses two anomaly score thresholds, which can be defined using the variables listed below:
+
+| Threshold                        | Variable                              |
+| -------------------------------- | ------------------------------------- |
+| Inbound anomaly score threshold  | `tx.inbound_anomaly_score_threshold`  |
+| Outbound anomaly score threshold | `tx.outbound_anomaly_score_threshold` |
+
+A simple way to set these thresholds is to uncomment and use rule 900110:
+
+```apache
+SecAction \
+ "id:900110,\
+  phase:1,\
+  nolog,\
+  pass,\
+  t:none,\
+  setvar:tx.inbound_anomaly_score_threshold=5,\
+  setvar:tx.outbound_anomaly_score_threshold=4"
+```
+
+### Severity Levels
+
+Each CRS rule has an associated *severity level*. Different severity levels have different anomaly scores associated with them. This means that different rules can increment the anomaly score by different amounts if the rules match.
+
+The four severity levels and their *default* anomaly scores are:
+
+| Severity Level | Default Anomaly Score |
+| -------------- | --------------------- |
+| **CRITICAL**   | 5                     |
+| **ERROR**      | 4                     |
+| **WARNING**    | 3                     |
+| **NOTICE**     | 2                     |
+
+For example, by default, a single matching `CRITICAL` rule would increase the anomaly score by 5, while a single matching `WARNING` rule would increase the anomaly score by 3.
+
+The default anomaly scores are usually acceptable. It is possible, however, to set custom anomaly scores for severity levels. To do so, uncomment rule 900100 and set the anomaly scores as desired:
```

Comment:
I would be even stronger here. Proposal: Rarely ever changed.

Reply:
Agreed. Amended.